README.md

```diff
@@ -160,3 +181,26 @@ Note: Downloading files is not recommended if your computer has access to the NG
 
 * Error: Requested resource wasn't found at http://irida.ca/api/...
   > The sample or project that you requested does not exist in the NGS Archive REST API.  Check your options for the project id (-p or --project) and sample id (-s or --sample) and try again.
+
+## Configuration file
+
+A configuration file, structured as shown below,
+can be used to feed URLs (`BASEURL` and `GALAXYURL`) and the Galaxy API key (`GALAXYAPIKEY`).
```

Comment:
I am curious why GALAXY and GALAXYAPIKEY were included in this example as I cannot find any references to it within the code.

Reply:
Will check, most likely it's something which can be zapped as only matters at QIB :) brb